src/triggered/wizards.ts

```diff
@@ -0,0 +1,195 @@
+import { html } from 'lit-element';
+import { get } from 'lit-translate';
+
+import '../filtered-list.js';
+
+import { List } from '@material/mwc-list';
+import { SingleSelectedEvent } from '@material/mwc-list/mwc-list-foundation';
+
+import {
+  createElement,
+  EditorAction,
+  newWizardEvent,
+  Wizard,
+  WizardAction,
+  WizardInput,
+} from '../foundation.js';
+import { clientIcon, controlBlockIcons, inputIcon } from '../icons.js';
+import {
+  Connection,
+  getConnection,
+  getConnectionIndexOf,
+} from './CommunicationMapping.js';
+
+function sinkPrimary(sink: Element): string {
+  const ln =
+    (sink.getAttribute('prefix') ?? '') +
+    sink.getAttribute('lnClass') +
+    (sink.getAttribute('lnInst') ?? '');
+
+  return sink.tagName === 'ClientLN'
+    ? ln
+    : ln +
+        '>' +
+        sink.getAttribute('doName') +
+        '.' +
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
        "." +
```